src/components/WalletInit/WalletInitScreen.js

```diff
@@ -36,12 +38,40 @@ const messages = defineMessages({
     id: 'components.walletinit.walletinitscreen.restoreWalletButton',
     defaultMessage: '!!!Restore wallet',
   },
+  restoreNormalWalletLabel: {
+    id: 'components.walletinit.walletinitscreen.restoreNormalWalletLabel',
+    defaultMessage: '!!!15-word Wallet',
+  },
+  restoreNormalWalletExplanation: {
+    id: 'components.walletinit.walletinitscreen.restoreNormalWalletExplanation',
+    defaultMessage:
+      '!!!If you have a Yoroi recovery phrase consisting of 15 ' +
+      'words generated when you created a Yoroi Wallet, choose this option ' +
+      'to restore your wallet.',
+  },
+  restore24WordWalletLabel: {
+    id: 'components.walletinit.walletinitscreen.restore24WordWalletLabel',
+    defaultMessage: '!!!24-word Wallet',
+  },
+  restore24WordWalletExplanation: {
+    id: 'components.walletinit.walletinitscreen.restore24WordWalletExplanation',
+    defaultMessage:
```

Comment:
It's probably better to make the use something like
```
If you have a recovery phrase consisting of {wordLength} words, choose this option to restore your wallet.
```
Since then you can re-use the same translation string for all languages (even if we add  support for 12-word and 18-word in the future)